neo/rawio/intanrawio.py

```diff
@@ -116,9 +116,13 @@ def _parse_header(self):
 
         filename = Path(self.filename)
 
+        # Input checks
         if not filename.exists() or not filename.is_file():
             raise FileNotFoundError(f"{filename} does not exist")
 
+        if not (self.filename.endswith(".rhd") or self.filename.endswith(".rhs")):
+            raise ValueError(f"{self.filename} is not a valid Intan file. Expected .rhd or .rhs extension")
+
```

Comment:
We sort of need both I guess, one tests the file exists and the other that is of the correct path. Otherwise a non-existen file that ends in rhs might cause a confusion error down the line, right?

Reply:
Okay. I guess we could also do your boolean and then add a quick exists(), but I think it is also fine to keep them as two separate checks as well.

Reply:
Mmm I think the two checks point the user to two different problems. I think they should remain separate. You mght have pass the wrong file or your might have passed other file from intan that is not rhs or rhd.  I think we should keep it more or less as it is now.